index.js

```diff
@@ -60,7 +64,11 @@ function config(opts) {
             cfg[name] = value;
         }
         else if ( opt.type === 'boolean' ) {
-            booleanValue = valid.boolean[value.toLowerCase()];
+            if ( typeOf(value) != 'boolean' ) {
```

Comment:
Oh ignore me, you have `typeOf` above. However, I think it's worth taking that out and just using `typeof` instead.